src/LiveComponent/src/DependencyInjection/LiveComponentExtension.php

```diff
@@ -89,7 +89,7 @@ public function load(array $configs, ContainerBuilder $container): void
             AsLiveComponent::class,
             function (ChildDefinition $definition, AsLiveComponent $attribute) {
                 $definition
-                    ->addTag('twig.component', array_filter($attribute->serviceConfig()))
+                    ->addTag('twig.component', array_filter($attribute->serviceConfig(), static fn ($v) => null !== $v))
```

Comment:
Is this related to this PR ?

Reply:
Without this, it does not work. It's a separate commit, so it can be cherry-picked as a bugfix or it can be moved to it's own PR if needed.

Reply:
Is there an error without it? I actually can't remember why we have that `array_filter()` at all

Reply:
If I remove the `array_filter` it breaks the compiler pass in TwigComponent...

https://github.com/symfony/ux/blob/a2418ef146f09a858bd12ed0116c3f8058d0a3a8/src/TwigComponent/src/DependencyInjection/Compiler/TwigComponentPass.php#L48

and other places where `null` is not checked.

Reply:
Yes but would that break things now that you allow empty strings ?

Reply:
I think it's not, all tests are green, but I can add `'' !== $v` too, if needed.

Reply:
Man it took me so much time to understand i'm sorry .. 

I suggest that we handle that in another PR, as it's clearly not in LiveComponent that we should handle this.

https://github.com/symfony/ux/pull/1402